solutions/devsprint-michel-ferreira-1/src/main/kotlin/com/devpass/challengehexagonal/resources/repository/database/client/TransactionSpringDataRepository.kt

```diff
@@ -2,8 +2,14 @@ package com.devpass.challengehexagonal.resources.repository.database.client
 
 import com.devpass.challengehexagonal.resources.repository.entity.TransactionEntity
 import org.springframework.data.repository.CrudRepository
+import java.time.LocalDateTime
 
 interface TransactionSpringDataRepository : CrudRepository<TransactionEntity, Long> {
 
     fun findByAccountId(accountId: Long): List<TransactionEntity>
+
```

Comment:
A nossa consulta vai precisar passar um range de data para que o SQL gere algo assim:
SELECT *
FROM xyz
WHERE abc BETWEEN CAST('2022-08-01' AS DATETIME) AND CAST('2022-08-02' AS DATETIME);